src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/NetherGoldPan.java

```diff
@@ -29,8 +29,13 @@ public NetherGoldPan(ItemGroup itemGroup, SlimefunItemStack item, RecipeType rec
     }
 
     @Override
-    public @Nonnull Material getInputMaterial() {
-        return Material.SOUL_SAND;
+    public @Nonnull Set<Material> getInputMaterials() {
+        Set<Material> materials = new HashSet<>();
+
+        materials.add(Material.SOUL_SAND);
+        materials.add(Material.SOUL_SOIL);
+
+        return materials;
     }
```

Comment:
It is not very efficient to create a new `HashSet` here everytime, make a private final field and just return that one here.